.github/workflows/linux-ppc64-cpu-gcc.yml

```diff
@@ -73,3 +73,95 @@ jobs:
         export PATH=$GITHUB_WORKSPACE/qemu-install/bin:$PATH
         cd build
         TESTS_EXECUTABLE_LOADER=qemu-ppc64le TESTS_EXECUTABLE_LOADER_ARGUMENTS="-L;/usr/powerpc64le-linux-gnu" ctest --output-on-failure -j 2
+  linux-gcc-power9le-vsx:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: cache-qemu
+      id: cache-qemu
+      uses: actions/cache@v3
+      with:
+        path: qemu-install
+        key: qemu-ppc64le-install-20220502-2
+    - name: install-qemu-build-deps
+      if: steps.cache-qemu.outputs.cache-hit != 'true'
+      run: |
+        sudo apt-get update
+        sudo apt-get install autoconf automake autotools-dev ninja-build
+    - name: checkout-qemu
+      if: steps.cache-qemu.outputs.cache-hit != 'true'
+      uses: actions/checkout@v3
+      with:
+        repository: qemu/qemu
+        path: qemu
+        ref: f5643914a9e8f79c606a76e6a9d7ea82a3fc3e65
+    - name: qemu
+      if: steps.cache-qemu.outputs.cache-hit != 'true'
+      run: |
+        cd qemu
+        ./configure --prefix=$GITHUB_WORKSPACE/qemu-install --target-list=ppc64le-linux-user --disable-system
+        make -j2
+        make install
+
+    - name: powerpc64le-gnu-toolchain
+      run: |
+        sudo apt-get update
+        sudo apt-get install g++-powerpc64le-linux-gnu
+
+    - name: configure
+      run: mkdir build && cd build && cmake -DCMAKE_TOOLCHAIN_FILE=../toolchains/power9le-linux-gnu-vsx.toolchain.cmake -DNCNN_BUILD_TOOLS=OFF -DNCNN_BUILD_EXAMPLES=OFF -DNCNN_BUILD_TESTS=ON ..
+    - name: build
+      run: cmake --build build -j 2
+
+    - name: test
+      run: |
+        export PATH=$GITHUB_WORKSPACE/qemu-install/bin:$PATH
+        cd build
+        TESTS_EXECUTABLE_LOADER=qemu-ppc64le TESTS_EXECUTABLE_LOADER_ARGUMENTS="-L;/usr/powerpc64le-linux-gnu" ctest --output-on-failure -j 2
+  linux-clang-power9le-vsx:
```

Comment:
The new ci is useful to help ensure future changes are powerpc vsx compatible !
But the clang ci cries, either fix it or drop the clang support

Reply:
I'm not certain why the Clang CI is failing; it might be due to an outdated Clang version. I've disabled that CI job so that it doesn't block a merge (I'll investigate later). Let me know if merging the Clang toolchain file must be accompanied by corresponding CI tests.

Reply:
It's this bug. https://github.com/llvm/llvm-project/issues/49864

Seems that LLVM 13.0 or higher is required to use SSE to VSX translation in C++ mode, and the CI environment is on LLVM 10.0. I could probably rig cmake to detect this brokenness and disable SSE in such cases, similar to how this PR already checks whether SSE4.1 to VSX translation is broken. That should allow us to re-enable the Clang CI build. Would that be OK?

Reply:
Actually, hmm, I think I'd prefer to do that as a follow-up PR if that's OK with you. Don't really want to increase the scope of this PR any further if it can be avoided.

Reply:
I think the requirement about clang version can be mentioned in build doc
The clang toolchain file itself can be added, without adding ci for the time being

Reply:
Added a note about this to the build doc.